src/styles/main.scss

```diff
@@ -128,10 +128,7 @@ $primary-disabled: lighten( map-get($theme-colors,"secondary"), 20% );
 }
 
 .panel-property {
-  border-color: #8C8A83;
-  border-style: solid;
-  border-width: 1px;
-  background-color: #F8F6EF;
+  background-color: #F7F4F1;
```

Comment:
Can we stick this color in a variable at the top of the file?

Reply:
done